pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -623,6 +623,15 @@ func ValidateVirtualMachineInstanceSpec(field *k8sfield.Path, spec *v1.VirtualMa
 			})
 		}
 
+		// Check interface name doesn't contain dots
+		if strings.Contains(iface.Name, ".") {
+			causes = append(causes, metav1.StatusCause{
+				Type:    metav1.CauseTypeFieldValueInvalid,
+				Message: fmt.Sprintf("Network interface name cannot contain \".\": %s", iface.Name),
```

Comment:
Once we see what is the cause of this issue, we can hopefully define what is the format of a valid network and if dot is the only invalid character.